.github/workflows/test.yml

```diff
@@ -43,6 +45,7 @@ jobs:
       with:
         name: earnestresearch-public
         signingKey: '${{ secrets.EARNESTRESEARCH_PUBLIC_CACHIX_SIGNING_KEY }}'
+    - run: cachix use pre-commit-hooks
```

Comment:
Will we want this one locally, or will CI get the relevant paths into earnestresearch-public?  cachix-action pushes new store paths, but I don't know conclusively whether that includes newly _downloaded_ paths.

Reply:
It includes downloaded paths too (it doesn't distinguish source, just that new derivations exist). So not needed elsewhere.

Reply:
Especially because pre-commit-hooks uses its own pkgs, seems good to reuse their cache :)